python/server/ServerWrapper.cpp

```diff
@@ -93,8 +93,25 @@ namespace {
     { return object(PathToString(GetResourceDir())); }
 
     // Wrapper for GetUserConfigDir
-    object GetUserConfigDirWrapper()
-    { return object(PathToString(GetUserConfigDir())); }
+    boost::python::str GetUserConfigDirWrapper()
+    { return boost::python::str(PathToString(GetUserConfigDir())); }
+
+    // Wrapper for GetUserDataDir
+    boost::python::str GetUserDataDirWrapper()
+    { return boost::python::str(PathToString(GetUserDataDir())); }
+
+    // Wrapper functions for OptionDB access
+    boost::python::object GetOptionsDBOptionStr(const std::string &option)
```

Comment:
`boost::python::object` could be used as an `object`. Also I think it better to use `str` same way.

Reply:
I only moved that code, didn't change it. But of course, it's probably better to use `boost::python::str`.